src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts

```diff
@@ -77,30 +65,32 @@ export const thorchainApi: Swapper2Api = {
     })
   },
 
+  // FIXME: implement in a way that doesn't cause circular dependencies
+  // eslint-disable-next-line require-await
   checkTradeStatus: async ({
     txId,
   }): Promise<{ status: TxStatus; buyTxId: string | undefined; message: string | undefined }> => {
-    const thorchainSwapper = new ThorchainSwapper()
+    // const thorchainSwapper = new ThorchainSwapper()
```

Comment:
We'll need to re-implement this in a way that doesn't cause circular dependancies.